saw-core/src/Verifier/SAW/SharedTerm.hs

```diff
@@ -1403,89 +1586,125 @@ scIntMod sc x y = scGlobalApply sc "Prelude.intMod" [x, y]
 scIntMin sc x y = scGlobalApply sc "Prelude.intMin" [x, y]
 scIntMax sc x y = scGlobalApply sc "Prelude.intMax" [x, y]
 
--- primitive intNeg/intAbs :: Integer -> Integer;
+-- | Create terms applying the (unary) integer primitives.
+--
+-- > intNeg/intAbs : Integer -> Integer;
```

Comment:
Same comment as above: The generated docs may be improved by splitting into separate type signatures.